src-plugins/medSegmentation/msegAlgorithmPaintToolbox.cpp

```diff
@@ -436,8 +439,8 @@ AlgorithmPaintToolbox::AlgorithmPaintToolbox(QWidget *parent ) :
     magicWandLayout = new QFormLayout(this);
     magicWandLayout->addRow(m_wandInfo);
     magicWandLayout->addRow(magicWandCheckboxes);
-    magicWandLayout->addRow(magicWandLayout1);
     magicWandLayout->addRow(magicWandLayout2);
+    magicWandLayout->addRow(magicWandLayout1);
```

Comment:
LowerThreshold above UpperThreshold, I guess we made the opposite change not that long ago. We should set that for good!
However I think it's better when LowerThresh is actually .... lower than UpperThr...